modules/core/src/lib/composite-layer.js

```diff
@@ -157,6 +157,18 @@ export default class CompositeLayer extends Layer {
       const overridingSublayerProps = overridingProps && overridingProps[sublayerProps.id];
       const overridingSublayerTriggers =
         overridingSublayerProps && overridingSublayerProps.updateTriggers;
+
+      if (overridingSublayerProps) {
+        const propTypes = this.constructor._propTypes;
+        for (const key in overridingSublayerProps) {
+          const propType = propTypes[key];
+          // eslint-disable-next-line
```

Comment:
Which rule needs to be ignored? `eslint-disable-next-line` ?

Reply:
max-depth

Reply:
Use `// eslint-disable-next-line max-depth`